cmd/suggestion/nas/enas/v1beta1/requirements.txt

```diff
@@ -1,4 +1,5 @@
 grpcio>=1.41.1
 googleapis-common-protos==1.6.0
 cython>=0.29.24
-tensorflow==2.11.0
+tensorflow==2.13.0
+protobuf<=3.20.3
```

Comment:
This requirement is for the reason the same as https://github.com/kubeflow/katib/pull/2140#issuecomment-1492417056.